components/ProgressView/examples/ProgressViewExample.m

```diff
@@ -119,45 +122,57 @@ - (void)viewDidLoad {
   self.navigationItem.rightBarButtonItem.accessibilityIdentifier = @"animate_button";
 }
 
+-(void)setupContainer {
+  self.container = [[UIView alloc] initWithFrame:self.view.bounds];
+  self.container.translatesAutoresizingMaskIntoConstraints = NO;
+  [self.view addSubview:self.container];
+  UILayoutGuide *layoutGuide = self.view.layoutMarginsGuide;
```

Comment:
This is iOS 9+, right? I don't think we have dropped iOS 8 yet, but I could easily be mistaken.